templates/data-sources/metal_connection.md.tmpl

```diff
@@ -56,3 +56,4 @@ In addition to all arguments above, the following attributes are exported:
   * `link_status` - Port link status.
   * `virtual_circuit_ids` - List of IDs of virtual cicruits attached to this port.
 * `token` - (Deprecated) Fabric Token required to configure the connection in Equinix Fabric with the [equinix_fabric_connection](../resources/fabric_connection.md) resource or from the [Equinix Fabric Portal](https://fabric.equinix.com/dashboard). If your organization already has connection service tokens enabled, use `service_tokens` instead.
+* `authorization_code` - Fabric Authorization Code to configure the NIMF connection in Equinix Fabric with the [equinix_fabric_connection](../resources/fabric_connection.md) resource or from the [Equinix Fabric Portal](https://fabric.equinix.com/dashboard).
```

Comment:
Instead of manually updating this I would recommend using `{{ .SchemaMarkdown }}` to replace the attributes list and have it build the correct attribute from the description in the schema.

Reply:
I have used {{ .SchemaMarkdown }} to replace attributes list but manually updated attribute reference for metal connection